adagucserverEC/CDataSource.cpp

```diff
@@ -1288,9 +1289,9 @@ CStyleConfiguration *CDataSource::getStyle(){
     }
   }
   
-    #ifdef CDATASOURCE_DEBUG      
-  CDBDebug("/getStyle");
-#endif
+//     #ifdef CDATASOURCE_DEBUG      
```

Comment:
Als dit wat nu in commentaar staat niet meer nodig is kan het het beste weggehaald worden.